specification/security/resource-manager/Microsoft.Security/preview/2025-07-01-preview/examples/DefenderForStorage/GetMalwareScan_example.json

```diff
@@ -0,0 +1,29 @@
+{
+  "parameters": {
+    "api-version": "2025-07-01-preview",
+    "resourceId": "subscriptions/20ff7fc3-e762-44dd-bd96-b71116dcdc23/resourceGroups/SampleRG/providers/Microsoft.Storage/storageAccounts/samplestorageaccount",
+    "settingName": "current",
+    "scanId": "latest"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "properties": {
+          "scanId": "0ce362c5-87a5-4030-ba6a-109566cd7b3d",
+          "scanStatus": "Completed",
+          "scanStartTime": "2025-07-12T12:20:14.6364816Z",
+          "scanEndTime": "2025-07-12T12:21:17.5648386Z",
+          "scanSummary": {
+            "blobs": {
+              "totalBlobsScanned": 40,
+              "maliciousBlobsCount": 10,
+              "skippedBlobsCount": 0,
```

Comment:
shouldn't failedBlobsCount always be present in the response too? just wondering!